src/patternfly/components/Popover/popover.scss

```diff
@@ -67,38 +94,90 @@
     .pf-c-popover__arrow {
       bottom: 0;
       left: 50%;
-      transform: translateX(var(--pf-c-popover__arrow--m-top--TranslateX)) translateY(var(--pf-c-popover__arrow--m-top--TranslateY)) rotate(var(--pf-c-popover__arrow--m-top--Rotate));
+      transform:
+        translateX(var(--pf-c-popover__arrow--m-top--TranslateX))
```

Comment:
We typically keep properties with multiple values on a single line, though if we do break them up into multiple lines, they're all on new lines. In this case, I would just put the `transform` functions on a single line.

Reply:
This applies to other instances in the stylesheet, so I won't comment on those.

Reply:
This is fixed now.